project/src/main/java/com/google/sps/data/User.java

```diff
@@ -0,0 +1,16 @@
+package com.google.sps.data;
+
+public final class User {
```

Comment:
Great User class! All of the parameters/naming clearly indicate what values are stored and used.